app/models/user.rb

```diff
@@ -7,7 +7,7 @@ class User < ApplicationRecord
   validates_confirmation_of :password
   has_many :orders
 
-
+  belongs_to :merchant, optional: true
```

Comment:
This is how we can connect a merchant employee(a user with a role: 1) to the merchant it works for

Reply:
Nice work